.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+
+go:
+- 1.9.*
```

Comment:
`x`, not `*`: https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use